### ISSUE TITLE
R4R pager is missing focus outlines

### ISSUE THREAD

**Issue by bennettcc** [NCIOCPL/cgov-digital-platform#3669]
### Basic Information ###
- Issue: no focus outline on the pager, search text field
- Desired Outcome:

### Steps to Reproduce ###
1. got to https://www-dev-acsf.cancer.gov/research/resources
2. click on Search filed - no outline
3. go to https://www-dev-acsf.cancer.gov/research/resources/search?from=0
4. click in any page in the pager - no blue outline on focus
### Screen Shots ###
_Add desired screen shots_
